ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFFromUnixTime.java

```diff
@@ -60,7 +62,7 @@
 
   private transient SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
   private transient String lastFormat = null;
-
+  private transient DateTimeFormatter FORMATTER =  DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
```

Comment:
Update this variable in configure method variable too.

Reply:
Done

Reply:
"yyyy" -> "uuuu"

Reply:
Done